src/components/Layout.tsx

```diff
@@ -63,99 +63,79 @@ const Layout = ({
+        active: <MaterialIcon icon="account_circle" type="filled" />,
       },
-      url: "/s/504/teachers",
+      url: "/account",
     },
   ];
   const teacherNav = [
     {
-      name: t("navigation.home"),
+      name: t("navigation.teach"),
       icon: {
-        inactive: <MaterialIcon icon="home" type="outlined" />,
-        active: <MaterialIcon icon="home" type="filled" />,
+        inactive: <MaterialIcon icon="school" type="outlined" />,
+        active: <MaterialIcon icon="school" type="filled" />,
       },
-      url: "/t/home",
+      url: "/teach",
     },
     {
-      name: t("navigation.schedule"),
+      name: t("navigation.class"),
       icon: {
-        inactive: <MaterialIcon icon="dashboard" type="outlined" />,
-        active: <MaterialIcon icon="dashboard" type="filled" />,
+        inactive: <MaterialIcon icon="groups" type="outlined" />,
+        active: <MaterialIcon icon="groups" type="filled" />,
       },
-      url: "/t/schedule",
+      url: "/class/504/manage",
     },
     {
-      name: t("navigation.subjects"),
+      name: t("navigation.news"),
       icon: {
-        inactive: <MaterialIcon icon="school" type="outlined" />,
-        active: <MaterialIcon icon="school" type="filled" />,
+        inactive: <MaterialIcon icon="newspaper" type="outlined" />,
+        active: <MaterialIcon icon="newspaper" type="filled" />,
       },
-      url: "/t/subjects/teaching",
+      url: "/news",
     },
     {
-      name: t("navigation.class"),
+      name: t("navigation.me"),
       icon: {
-        inactive: <MaterialIcon icon="groups" type="outlined" />,
-        active: <MaterialIcon icon="groups" type="filled" />,
+        inactive: <MaterialIcon icon="account_circle" type="outlined" />,
+        active: <MaterialIcon icon="account_circle" type="filled" />,
       },
-      url: "/t/202/class",
+      url: "/account",
     },
   ];
-  const newsNavItem = {
-    name: t("navigation.news"),
-    icon: {
-      inactive: <MaterialIcon icon="newspaper" type="outlined" />,
-      active: <MaterialIcon icon="newspaper" type="filled" />,
-    },
-    url: "/news",
-  };
-  const adminNavItem = {
-    name: t("navigation.admin"),
-    icon: {
-      inactive: <MaterialIcon icon="security" type="outlined" />,
-      active: <MaterialIcon icon="security" type="filled" />,
-    },
-    url: "/t/admin",
-  };
 
   useEffect(() => {
     const role = session?.user?.user_metadata.role;
-    const isAdmin = session?.user?.user_metadata.isAdmin;
 
     // Decide the Navigation the user is going to see based on their role
-    // Append the Admin Nav Item to the Navigation if the user is an admin
-    if (role == "student")
-      setNavItems([...studentNav, isAdmin ? adminNavItem : newsNavItem]);
-    else if (role == "teacher")
-      setNavItems([...teacherNav, isAdmin ? adminNavItem : newsNavItem]);
+    if (role == "student") setNavItems(studentNav);
```

Comment:
where did the admin part go?

Reply:
Seems like I wasn’t done working on Navigation when I made the PR. Oh well, things slip through the cracks.

Anywho, if you recall from the sitemap diagram I made a while ago, I was going to merge Me and Admin, and that’s why Admin shouldn’t be there. 

I’m thinking, though, if Admin should be back on Navigation. After all, an admin would probably go to Admin enough to warrant one. Thoughts?